examples/PhasorDynamics/Tiny/ThreeBus/Classical/ThreeBusClassical.cpp

```diff
@@ -177,6 +177,9 @@ int main()
 
   SystemModel<scalar_type, index_type> sys(data);
   sys.allocate();
+  sys.initialize();
+  sys.evaluateResidual();
+  sys.evaluateJacobian();
```

Comment:
These calls should not be made here. Evaluating residual and Jacobian should be done by the numerical integrator only, not a user code.

```suggestion
```